### ISSUE TITLE
Cannot update textStyle in tooltip with formatter

### ISSUE THREAD

**Issue by pavelbabenko** [wuba/wrn-echarts#15]
Hi! I'm trying to customize tooltip in series with `type: 'custom'` using formatted function, but I'm not able to change styles of text

        type: 'custom',
        data: [7000],
        tooltip: {
          formatter: (params: CallbackDataParams) => {
            return `Data: ${params.data}`;
          },
          textStyle: {
            fontSize: 20,
          },
        },

### The result:
<img width="75" alt="image" src="https://user-images.githubusercontent.com/44643721/216325273-5ee9babc-a3cf-4e05-92ae-8bbca2e73d45.png">

**Comment by zhiqingchen** [comment]
Yes, this is a problem. tooltip in echarts uses html instead of svg, and html is not currently supported in wrn-echarts.

Alternatively, you can select svg rendering in the chart [editor in echarts](https://echarts.apache.org/examples/en/editor.html?c=line-sections) and locate if htm is used by the browser's review element.

We are also in the process of sorting out the relevant currently unsupported features.

**Comment by chj-damon** [comment]
is it possible to support this?

**Comment by zhiqingchen** [comment]
we will try it.

**Comment by zhiqingchen** [comment]
> custom tooltip in echarts uses html instead of svg, and html is not currently supported in wrn-echarts.

This sentence is misrepresented

**Comment by zhiqingchen** [comment]
https://github.com/apache/echarts/issues/18339

It may be an issue of echarts.
